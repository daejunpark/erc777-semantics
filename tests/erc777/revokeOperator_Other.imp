test.preamble();

// Testing revokeOperator(Operator) when Operator is not an operator
// A token holder MAY revoke an already revoked operator. A RevokedOperator MUST be emitted each time.
// should pass

test.revokeOperator(1);
