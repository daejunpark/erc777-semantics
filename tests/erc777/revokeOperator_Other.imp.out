7 is revoking operator 1
